model/src/main/resources/reference.yml

```diff
@@ -30,6 +30,13 @@ ort:
   enableRepositoryPackageCurations: true
   enableRepositoryPackageConfigurations: true
 
+  variablesToPropagate:
+    - HOME
+    - JAVA_HOME
+  additionalVariablesToPropagate:
+    - BUILD_PATH
+    - FAST_MODE
+
```

Comment:
Should we also propagate  `PATH` ?

When I run the Analyzer locally, I often run it with a modified `PATH` for it to find the different tools.

Reply:
We should also propagate `USER`, `PWD` and `LANG`.